hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml

```diff
@@ -45,4 +45,5 @@
   <suppress checks="IllegalImport" message="org\.apache\.htrace\.core"/>
   <suppress checks="ImportOrder" message="Extra separation in import group before"/>
   <suppress checks="MethodLength" files="DemoClient.java"/>
+  <suppress checks="LineLength" files="MultiTableSnapshotInputFormat.java"/>
```

Comment:
Could you please check if it is possible to add the package name to the file suppression?

Reply:
I have checked and it is possible. Please find the latest commit.

Reply:
Is it possible to suppress LineLenght check only on the offending lines in the javadoc instead of completely ignoring it in the org.apache.hadoop.hbase.mapred package?

https://stackoverflow.com/a/30070188/3521345

Reply:
Thanks i will look into it.

Reply:
I have suppressed LineLenghth check only on the offending lines in the javadoc using "@SuppressWarnings"  annotation. Kindly check the 4th commit for this change.